app/containers/ItunesContainer/reducer.js

```diff
@@ -0,0 +1,49 @@
+/*
+ *
+ * itunesContainer reducer
+ *
+ */
+
+import produce from 'immer';
+import { createActions } from 'reduxsauce';
+
+export const initialState = {
+  searchTerm: null,
+  gridData: {},
+  searchError: null
+};
+
+export const { Types: itunesContainerTypes, Creators: itunesContainerCreators } = createActions({
+  searchItunes: ['searchTerm'],
+  successSearchItunes: ['data'],
+  failureSearchItunes: ['error'],
+  clearGridData: {}
+});
+
+/* eslint-disable default-case, no-param-reassign */
+export const itunesContainerReducer = (state = initialState, action) =>
+  produce(state, (draft) => {
+    switch (action.type) {
+      case itunesContainerTypes.SEARCH_ITUNES:
```

Comment:
If you will make the change as mentioned in the above comment
Then
REQUEST_GET_TRACKS is what this will change to

Reply:
updated the changes accordingly